atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -146,14 +146,18 @@ def __init__(self, tasker, workflow,
         }
         self.url = get_openshift(self.workflow, self.openshift_fallback)['url']
 
+        self.koji_task_id = None
         try:
             metadata = get_build_json().get("metadata", {})
-            self.koji_task_id = int(metadata['labels']['koji-task-id'])
+            koji_task_id = metadata['labels'].get('koji-task-id')
```

Comment:
:taco:  FINALLY we'll stop seeing those tracebacks in the osbs-client integration tests!